esmvaltool/cmorizers/obs/cmorize_obs_cds_satellite_lai_fapar.py

```diff
@@ -0,0 +1,220 @@
+"""ESMValTool CMORizer for cds-satellite-lai-fapar data.
+
+Tier
+   Tier 3
+Source
+   https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-lai-fapar?tab=form
+Last access
+   20190703
+
+Download and processing instructions
+   - Open in a browser the data source as specified above
+   - Put the right ticks:
+      - Tick variables FAPAR and LAI
+      - Tick SPOT
+      - Tick V1
+      - Tick all available years
+      - Tick all available months
+      - Tick Nominal day 20
+   - Click 'submit form'
+   - According to ESMValTool practice, put them in the right rawobsdir folder
+
+Notes
+-----
+   - This script regrids and cmorizes the above dataset.
+
+Caveats
+   - Fails setting standard name for variable FAPAR
+
+Modification history
+   20190703-A_crez_ba: written.
+"""
+
+import glob
+import logging
+import os
+from copy import deepcopy
+from datetime import datetime
+from warnings import catch_warnings, filterwarnings
+
+import cf_units
+import iris
+
+from esmvalcore.preprocessor import regrid
+from esmvaltool.cmorizers.obs import utilities as utils
+
+logger = logging.getLogger(__name__)
+
+
+def _attrs_are_the_same(cubelist):
+    # assume they are the same
+    attrs_the_same = True
+    allattrs = cubelist[0].attributes
+    for key in allattrs:
+        try:
+            unique_attr_vals = {cube.attributes[key] for cube in cubelist}
+        # This exception is needed for valid_range, which is an
+        # array and therefore not hashable
+        except TypeError:
+            unique_attr_vals = {tuple(cube.attributes[key])
+                                for cube in cubelist}
+        if len(unique_attr_vals) > 1:
+            attrs_the_same = False
+            print("Different values found for {0}-attribute: {1}"
+                  .format(key, unique_attr_vals))
+    return attrs_the_same
+
+
+def _cmorize_dataset(in_file, var, cfg, out_dir):
+    logger.info("CMORizing variable '%s' from input file '%s'",
+                var['short_name'], in_file)
+    attributes = deepcopy(cfg['attributes'])
+    attributes['mip'] = var['mip']
+
+    cmor_table = cfg['cmor_table']
+    definition = cmor_table.get_variable(var['mip'], var['short_name'])
+
+    cube = iris.load_cube(
+        str(in_file),
+        constraint=utils.var_name_constraint(var['raw']))
+
+    # Set correct names
+    cube.var_name = definition.short_name
+    try:
```

Comment:
I would replace this by something like:
```python
if definition.standard_name:
    cube.standard_name = definition.standard_name
```
that will not generate warnings where they are expected.